astroquery/simbad/core.py

```diff
@@ -309,10 +309,13 @@ def list_votable_fields(self):
 
         if not hasattr(self,'_votable_fields_table'):
             # load the table
-            votable_fields_table = Table.read(
-                get_pkg_data_filename(os.path.join('data',
-                                                   'votable_fields_table.txt')),
-                format='ascii')
+            dict_file = get_pkg_data_filename(
+                os.path.join('data', 'votable_fields_dict.json'))
+            with open(dict_file, "r") as f:
+                fields_dict = json.load(f)
+            rows_to_add = list(zip(*[iter(sorted(list(fields_dict.keys())))]))
```

Comment:
I'm not sure why we need the Table at all, this function could return the list as is. See similar functions in other modules, e.g. Eso.list_surveys(), etc.